README.md

```diff
@@ -104,8 +104,6 @@ has instructions for running the samples.
 | Sample                      | Source Code                       | Try it |
 | --------------------------- | --------------------------------- | ------ |
 | Quickstart | [source code](https://github.com/googleapis/nodejs-pubsub/blob/master/samples/quickstart.js) | [![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-pubsub&page=editor&open_in_editor=samples/quickstart.js,samples/README.md) |
-| Subscriptions | [source code](https://github.com/googleapis/nodejs-pubsub/blob/master/samples/subscriptions.js) | [![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-pubsub&page=editor&open_in_editor=samples/subscriptions.js,samples/README.md) |
```

Comment:
I'm surprised the list here and in `samples/README.md` went down, since those should be generated.  Can you install and try running `synthtool` to get the README generation stuff kickin?  
https://github.com/googleapis/synthtool/

Reply:
It looks like I had the wrong version of synthtool, and it wasn't finding the samples because they were in subdirs. I just moved them up, and it's working now.